packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -459,11 +565,11 @@ class _RenderableTileLayer extends _RenderableLayer<TileLayer> {
         //  * stagger: Hexagonal tiles move down or right by only a fraction,
         //             specifically 3/4 the width or height, for packing.
         if (_map.staggerAxis == StaggerAxis.y) {
-          offsetX = tx * _map.tileWidth + staggerX + halfDestinationTile.x;
```

Comment:
This was a bug for scaled hexagonal tiles. Fixed here and not copy-pastaed to isometric staggered.

Reply:
> Good stuff as always! Could you add an image to the PR description with an example of the outcome when using this?

will do - It seems I actually need to revisit the painting order in X stagger order - if we walk the traditional row/col,
3D like tiles are painted col-first, leading to ugly overlaps.